charts/harvester-network-controller/templates/rbac.yaml

```diff
@@ -82,3 +85,59 @@ subjects:
   - kind: ServiceAccount
     name: harvester-network-helper
     namespace: {{ .Release.Namespace }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: harvester-network-webhook
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "harvester-network-controller.labels" . | nindent 4 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: harvester-network-webhook
+rules:
+  - apiGroups: [ "" ]
+    resources: [ "secrets", "configmaps" ]
+    verbs: [ "*" ]
```

Comment:
may consider merging webhook RBAC with the network as one, or u should remove those unneeded permissions.

Reply:
They are all needed.